test/utils.dart

```diff
@@ -66,6 +67,40 @@ class _Parse extends Matcher {
   }
 }
 
+/// A matcher that validates the body of a multipart request after finalization.
+/// The string "{{boundary}}" in [pattern] will be replaced by the boundary
+/// string for the request, and LF newlines will be replaced with CRLF.
+/// Indentation will be normalized.
+Matcher bodyMatches(String pattern) => new _BodyMatches(pattern);
```

Comment:
This name is pretty generic for something that's specifically designed to work with multipart requests. Maybe `multipartBodyMatches()`?